pkgs/by-name/ch/chart-releaser/package.nix

```
@@ -0,0 +1,67 @@
+{ buildGoModule
+, coreutils
+, fetchFromGitHub
+, git
+, installShellFiles
+, kubernetes-helm
+, lib
+, makeWrapper
+}:
+
+buildGoModule rec {
+  pname = "chart-releaser";
+  version = "1.6.1";
+
+  # 0102fa30dd78df0d0f8093c4607e6080972ff82d
+
+  src = fetchFromGitHub {
+    owner = "helm";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-8+O9JErEB1Z/zlrWm975v5Qf0YG0lbPcjY5LlDKw8U4=";
+
+  };
+
+  # Don't run tests.
+  doCheck = false;
+  doInstallCheck = false;
+
+  vendorHash = "sha256-S/V1kTgD/cXaJNYpPPNjM9ya2zv6Bsy/YBn7I/1EjwI=";
+
+  postPatch = ''
+    substituteInPlace pkg/config/config.go \
+      --replace "\"/etc/ct\"," "\"$out/etc/ct\","
+  '';
+
+  # https://github.com/helm/chart-releaser/blob/fa01315c4668d4fca627a5afc67409e31b27305c/.goreleaser.yml#L37
+  ldflags = [
+    "-w"
+    "-s"
+    "-X github.com/helm/chart-releaser/v3/ct/cmd.Version=${version}"
+    "-X github.com/helm/chart-releaser/v3/ct/cmd.GitCommit=${src.rev}"
+    "-X github.com/helm/chart-releaser/v3/ct/cmd.BuildDate=19700101-00:00:00"
+  ];
+
+  nativeBuildInputs = [ installShellFiles makeWrapper ];
+
+  postInstall = ''
+    installShellCompletion --cmd cr \
+      --bash <($out/bin/ct completion bash) \
+      --zsh <($out/bin/ct completion zsh) \
+      --fish <($out/bin/ct completion fish) \
```

> /nix/store/wq046wv3s3h4m2n9ak3j534di2pzra93-stdenv-linux/setup: line 114: /nix/store/asvv4qv70amf2bd4wfqi2z8j8qih6n3m-chart-releaser-1.6.1/bin/ct: No such file or directory
/nix/store/wq046wv3s3h4m2n9ak3j534di2pzra93-stdenv-linux/setup: line 114: /nix/store/asvv4qv70amf2bd4wfqi2z8j8qih6n3m-chart-releaser-1.6.1/bin/ct: No such file or directory
/nix/store/wq046wv3s3h4m2n9ak3j534di2pzra93-stdenv-linux/setup: line 114: /nix/store/asvv4qv70amf2bd4wfqi2z8j8qih6n3m-chart-releaser-1.6.1/bin/ct: No such file or directory


```suggestion
    installShellCompletion --cmd cr \
      --bash <($out/bin/cr completion bash) \
      --zsh <($out/bin/cr completion zsh) \
      --fish <($out/bin/cr completion fish) \
```